test-suite/java/CMakeLists.txt

```diff
@@ -37,7 +37,7 @@ file(
   ../generated-src/cpp/*.cpp
   ../handwritten-src/cpp/*.cpp)
 
-set(test_suite_common_flags "-g -Wall -Werror -std=c++11")
+set(test_suite_common_flags "-g -Wall -Werror -std=c++14")
```

Comment:
This should remain at c++11. Djinni requires C++11 as a minimum, and shouldn't require C++14.